build/update-attribution-files/make_attribution.sh

```diff
@@ -63,17 +63,19 @@ else
             else
                 # if the git_tags match across release branches, save the output state to avoid
                 # rebuilding/regenerating
-                if [[ $TARGET == *"checksums"* ]]; then
-                    echo "Copying $LAST_RELEASE_BRANCH CHECKSUMS to $release"
-                    mkdir -p $PROJECT_ROOT/_output/$release
-                    sed "s/$LAST_RELEASE_BRANCH/$release/" $PROJECT_ROOT/$LAST_RELEASE_BRANCH/CHECKSUMS > $PROJECT_ROOT/$release/CHECKSUMS
-                fi
+                if [[ $LAST_RELEASE_BRANCH != "1-23" ]]; then
```

Comment:
i *think* instead of doing this here, you need to change line 59 to

` if [[ $LAST_RELEASE_BRANCH = "1-23" ]] || [[ "$GIT_TAG" != "$LAST_GIT_TAG"  || $TARGET == "update-go-mods" ]]; then`